packages/admin-next/dashboard/src/routes/inventory/inventory-detail/components/inventory-item-general-section.tsx

```diff
@@ -9,21 +9,14 @@ import { HttpTypes } from "@medusajs/types"
 
 type InventoryItemGeneralSectionProps = {
   inventoryItem: InventoryItemRes["inventory_item"] & {
-    variant: HttpTypes.AdminProductVariant | HttpTypes.AdminProductVariant[]
+    variants: HttpTypes.AdminProductVariant[]
   }
 }
 export const InventoryItemGeneralSection = ({
   inventoryItem,
 }: InventoryItemGeneralSectionProps) => {
   const { t } = useTranslation()
-
-  const variantArray = inventoryItem.variant
-    ? Array.isArray(inventoryItem.variant)
-      ? inventoryItem.variant
-      : [inventoryItem.variant]
-    : []
-
-  const variantTitles = variantArray.map((variant) => variant.title)
+  const variantTitles = inventoryItem.variants?.map((v) => v.title)
```

Comment:
comment: we should probably remove this row now since we have `Associated variants` section

Reply:
Makes sense, let me do that 👍